docs/logs/discovery/tailing-containers.md

```diff
@@ -0,0 +1,18 @@
+title: Shipping Container Logs
+description: Shipping Container Log Sources
+
+To enable automatic container logs shipping, start by clicking on the `Set Up` button. This will open the following flyout panel that allows configuring container log shipping:
```

Comment:
> start by clicking on the `Set Up` button. 

Hm, I don't know which Set Up button. :)  I happen to know it's on another page in the docs, but as a user I am not reading N pages of docs in sequence.  I think we should insert that image here.

the following flyout panel that allows configuring container log shipping
=> 
a flyout where you can configure container log shipping

Reply:
Adjusted.

![container-logs-docs](https://user-images.githubusercontent.com/11174375/100226517-28820280-2f20-11eb-9038-54cf5be27461.png)